common/src/main/java/com/graphicsfuzz/common/transformreduce/GlslShaderJob.java

```diff
@@ -106,12 +110,10 @@ public void makeUniformBindings() {
         if (decl instanceof VariablesDeclaration
             && ((VariablesDeclaration) decl).getBaseType().hasQualifier(TypeQualifier.UNIFORM)) {
           final VariablesDeclaration variablesDeclaration = (VariablesDeclaration) decl;
-          final QualifiedType qualifiedType = (QualifiedType) variablesDeclaration.getBaseType();
-          // We cannot yet deal with adding bindings for uniform arrays.
-          assert !(qualifiedType.getWithoutQualifiers() instanceof ArrayType);
+          // We cannot yet deal with uniforms with array base types (only array information provided
+          // with the variable's declaration info).
+          assert !(variablesDeclaration.getBaseType().getWithoutQualifiers() instanceof ArrayType);
```

Comment:
Could you add an example to explain what this means?

Reply:
Done.